src/modules/reviews/InformationSection/InformationCard/InformationModal.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { Modal } from "@/common/components/Modal";
+import { type Courses } from "@prisma/client";
+import { informationCardTheme } from "./InformationCard.theme";
+import { Button } from "@/common/components/Button";
+
+export const InformationModal = ({
+  course,
+}: {
+  course: Courses;
+}) => {
```

Comment:
issue: since this is a client component, it would not be a good idea to pass the whole course object as prop to this client component since we're only 2 props from course object

passing the whole course object will allow the client access to all the props of course object, not just those that are used in this component (description & name)

see [Component Level Data Access](https://nextjs.org/blog/security-nextjs-server-components-actions#component-level-data-access) by vercel on nextjs security practices for more detailed explanation


```suggestion
export const InformationModal = ({
  courseName,
  courseDesc,
}: {
  courseName: string
  courseDesc: string;
}) => {
```

Reply:
Fixed together with https://github.com/AfterClass-io/afterclass.io-v2/pull/146#discussion_r1618060124 at [eee3712](https://github.com/AfterClass-io/afterclass.io-v2/pull/146/commits/eee3712add2ae853732967c804deb14c766ebdda)